### ISSUE TITLE
CVE on path-to-regex

### ISSUE THREAD

**Issue by L4ngu0r** [DataDog/dd-trace-js#4665]
Hello, our audit reported a CVE on `path-to-regex`, can you update this deps? It is known on your side?

https://github.com/advisories/GHSA-9wv6-86v2-598j

```
┌─────────────────────┬────────────────────────────────────────────────────────┐
│ high                │ path-to-regexp outputs backtracking regular            │
│                     │ expressions                                            │
├─────────────────────┼────────────────────────────────────────────────────────┤
│ Package             │ path-to-regexp                                         │
├─────────────────────┼────────────────────────────────────────────────────────┤
│ Vulnerable versions │ <0.1.10                                                │
├─────────────────────┼────────────────────────────────────────────────────────┤
│ Patched versions    │ >=0.1.10                                               │
├─────────────────────┼────────────────────────────────────────────────────────┤
│ Paths               │ apps/front > dd-trace@5.17.0 > path-to-regexp@0.1.7    │
├─────────────────────┼────────────────────────────────────────────────────────┤
│ More info           │ https://github.com/advisories/GHSA-9wv6-86v2-598j      │
└─────────────────────┴────────────────────────────────────────────────────────┘
```

**Comment by BellaMay95** [comment]
Following. Looks like the version got bumped here just a little while ago so hopefully the new version gets released here soon. :)

**Comment by L4ngu0r** [comment]
PR merged https://github.com/DataDog/dd-trace-js/pull/4664 waiting for a release

**Comment by LucasHaddad** [comment]
is it being released soon? anything we can help with in this matter?

**Comment by thiagoribeir015** [comment]
Hi, any updates on a release here @L4ngu0r? 🙇🏻

**Comment by L4ngu0r** [comment]
@thiagoribeir015  I'm not a maintainer here :-) just a user waiting for them to push a new release

**Comment by JasonKleban** [comment]
bump

**Comment by tlhunter** [comment]
This should be available in `v5.23.0` and `v4.47.0`. Is anyone here still having an issue?

**Comment by L4ngu0r** [comment]
Fix on our side with the `v5.23.0` thanks 🙏
